### PR TITLE
Remove index

### PR DESCRIPTION
**Pull request by kingbrendann**

### CODE REVIEW

File: More2Life/FeedViewController.swift [hunk 1]
```diff
@@ -37,7 +37,7 @@ class FeedViewController: UIViewController {
             
             let fetchRequest: NSFetchRequest<FeedItem> = FeedItem.fetchRequest()
             fetchRequest.predicate = NSPredicate(format: "%K == true", #keyPath(FeedItem.isActive))
-            fetchRequest.sortDescriptors = [NSSortDescriptor(key: #keyPath(FeedItem.index), ascending: false)]
+            fetchRequest.sortDescriptors = [NSSortDescriptor(key: #keyPath(FeedItem.identifier), ascending: false)]
```
Comment (PorterHoskins) [inline More2Life/FeedViewController.swift]: This is weird. We won't be able to change to order of things.

File: More2Life/ImagePageViewController.swift [hunk 2]
```diff
@@ -51,7 +51,7 @@ extension ImagePageViewController: UIPageViewControllerDataSource {
     // MARK: - Page Indicator
     
     func presentationCount(for pageViewController: UIPageViewController) -> Int {
-        return imageViewControllers.count
+        return imageViewControllers.count > 1 ? 0 : -1
```
Comment (PorterHoskins) [inline More2Life/ImagePageViewController.swift]: What is the idea behind this?

File: More2Life/FeedViewController.swift (hunk 1 above)
Comment (kingbrendann) [reply More2Life/FeedViewController.swift]: Derp. I was thinking the identifier is a number but it is not - it's a string. This won't work here. 

Does a date work as a sort descriptor? If we added a createdAt date to all the feed items could we use that here?

File: More2Life/ImagePageViewController.swift (hunk 2 above)
Comment (kingbrendann) [reply More2Life/ImagePageViewController.swift]: this removes the page indicator if the page view has only one page. no need for the dot if it's by itself.

File: More2Life/FeedViewController.swift (hunk 1 above)
Comment (PorterHoskins) [reply More2Life/FeedViewController.swift]: Yes. That would work great.

File: More2Life/ImagePageViewController.swift (hunk 2 above)
Comment (PorterHoskins) [reply More2Life/ImagePageViewController.swift]: Isn't this correct logic?

```
imageViewControllers.count > 1 ? imageViewControllers.count : -1
```

Comment (kingbrendann) [reply More2Life/ImagePageViewController.swift]: duh

**Comment by kingbrendann** [comment]
Fixed both of those

**Review by PorterHoskins — APPROVED** [review]

**Pull request merged**